### ISSUE TITLE
Doll house picture

### ISSUE THREAD

**Issue by haizaar** [haizaar/doll-house-lights#4]
This is a reference issue to picture the house and list its dimensions. 

I marked approximate desired location of the leds.

It's 73cm wide, 120cm tall, and 34cm deep (w/o basement). Basement is hollow, with access latches on each side and has plenty of space to hine controller and powerbank.

![image](https://user-images.githubusercontent.com/58201/60228536-4f28ad80-98d6-11e9-8650-e2a6601903c1.png)

**Comment by jim-sokoloff** [comment]
Additional thought: we might be able to make a flickering orange effect for the fireplace and/or a flickering TV per the EEVBlog suggestion.

**Comment by jim-sokoloff** [comment]
You might run one channel to the "dining room", one channel to the "living room", one channel to the bath, right cornice light, and right attic light. (then that same, or a different channel) to the corresponding left side lights.

We can adjust the order of the lights in software, so make the runs however you like.

Might want to hide the IR receiver in the foundation wall in front [so it's easy to hit from anywhere], in the chimney, or under the eaves.

**Comment by jim-sokoloff** [comment]
Confirm size. That seems **huge**.

If it's as big as you indicate, you might consider running an intact strand of the LEDs [on the flex PCB] under the eaves. We could then do all kinds of effects there [chasing lights, christmas lights, a night light for the room].

Same thing inside the rooms; might use more than one LED in some of the rooms. (The wiring complexity is in the "in and out" and less within a given room.)

**Comment by haizaar** [comment]
The size is correct indeed: **73cm wide, 120cm tall, and 34cm deep** 

> you might consider running an intact strand of the LEDs [on the flex PCB] under the eaves. 

Good idea actually. I ordered 60led/m strip, but I think for eaves it will be too dense - I'm not sure what is the _minimum_ level of light a led con produce. So may be it worth ordering 30led/m strip as well. **What do you think?** (spacing them out using solder and wire seems like quite a bit of work).

> Same thing inside the rooms; might use more than one LED in some of the rooms. (The wiring complexity is in the "in and out" and less within a given room.)

Agree. I though about that as well. I think once I have a first assembly, I'll dry-fit it and try both in day and night to see whether it produces enough luminance.

**Comment by jim-sokoloff** [comment]
Well, you could use the 60 LED/m strips and, because each LED is individually addressable/controllable, you can make a 60 LED/m strip behave like a 30 LED/m strip...

Surely don't space them out with solder and wire, but if you have enough overall length of 60/m strips, I'd just plan on using what you have and handling it in software.

**Comment by haizaar** [comment]
Good idea about utilizing only every second led or so. Lets see what I end up with.

**Comment by jim-sokoloff** [comment]
As I was fooling around with the code, I also realized that you could also just run the lights on that strand at half intensity, giving an even light without burning too much power.
